src/main/views/courts/tabs/generalContent.njk

```diff
@@ -129,8 +126,7 @@
   </span>
   <span class="govuk-visually-hidden">{{mceMsg}}</span>
   <textarea class="govuk-textarea urgent-notice-rich-editor" id="urgent-notice" name="alert"
```

Comment:
Do we need "urgent-notice-rich-editor" etc now that we are no longer using tinymce? I think they were only used for that